mkdocs.yaml

```diff
@@ -17,10 +17,10 @@
 docs_dir: user_docs
 
 # Project information
-site_name: GHGA User Documentation
+site_name: User Documentation
```

Comment:
I removed it because it reads double with the logo, have you checked how it looks when it's rendered? Will be even stronger with the slimmer logo you suggested. I'm fine with either, just wanted to verify